### PR TITLE
Add ability to snooze Uphold Emails

### LINKED ISSUE #2537: Email changes
- [ ] KYC on-boarding email down to 1 year
- [ ] Add link in email to support page (Albert to sync up with support-team)
- [ ] Tell users how much we owe them during the delete flow (We need to figure out how to store this info. Accounting guidance is good) (Albert will chase down and it's pending discussion)

### PR DESCRIPTION
**Pull request by corymcdonald**
## Add ability to snooze Uphold Emails

Closes #2537

#### Features

- Sends emails once per year to non-kyc'd publishers
- Allow unsubscribe of uphold payment reminder emails

### CODE REVIEW

File: app/controllers/publishers/uphold_controller.rb [hunk 1]
```diff
@@ -85,6 +85,13 @@ def create
       redirect_to(home_publishers_path, alert: t(".uphold_error", message: e.message))
     end
 
+    def update
+      send_emails = params[:uphold_connection_send_emails]
+      # If the value is blank then user has opted out of emails.
```
Comment (yachtcaptain23) [inline app/controllers/publishers/uphold_controller.rb]: IMO you should make it 100 years from now (or some other unreasonable date in the future). If the value is blank, then that means the user hasn't been sent emails

File: db/migrate/20200210213507_add_send_emails_to_uphold_connection.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class AddSendEmailsToUpholdConnection < ActiveRecord::Migration[6.0]
+  def change
+    add_column :uphold_connections, :send_emails, :datetime, default: -> { 'CURRENT_TIMESTAMP' }
+  end
+
+  # If send_emails is blank then the user has opted out
+  UpholdConnection.update(send_emails: DateTime.now)
```
Comment (yachtcaptain23) [inline db/migrate/20200210213507_add_send_emails_to_uphold_connection.rb]: Unused code?

Comment (corymcdonald) [reply db/migrate/20200210213507_add_send_emails_to_uphold_connection.rb]: https://github.com/brave-intl/publishers/pull/2545/commits/06c67064f71662c3644df540cb66ddcaf433594f

File: app/views/publishers/settings/index.html.slim [hunk 3]
```diff
@@ -17,9 +17,26 @@
...
 
+.single-panel--wrapper.single-panel--wrapper--large.single-panel--wrapper--short.mb-4
+  .single-panel--padded-content--short-padding#publishers_contact
+    h5 Email Settings
     = form_for(@publisher, url: publishers_path, html: { id: "update_publisher_marketing_email_form", class: 'mt-4 ml-4'}) do |f|
       = f.check_box(:subscribed_to_marketing_emails, class: "form-check-input")
-      = f.label(:marketing_emails, class: "form-check-label", for: "marketing_emails")
+      = f.label(:marketing_emails, class: "form-check-label", for: "publisher_subscribed_to_marketing_emails")
+
+    - unless @publisher.paypal_locale?(params[:locale])
```
Comment (yachtcaptain23) [inline app/views/publishers/settings/index.html.slim]:
```suggestion
    - if @publisher.uphold_connection.present? && !@publisher.paypal_locale?(params[:locale])
```

**Pull request merged**